fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DimensionsServlet.java

```diff
@@ -332,6 +326,27 @@ public Response getDimensionRows(
         }
     }
 
+    /***
+     * Get the pagination of dimension rows.
+     *
+     * @param apiRequest The apiRequest
+     * @param searchProvider The searchProvider
+     * @param paginationParameters The pagination parameters
+     * @return pagination of dimensionRow
+     */
+    protected Pagination<DimensionRow> getPagedRows(
```

Comment:
For public `getDimensionRows()` which invokes this method, the documentation should indicate this overridable method is invoked. By convention, a method that invokes overridable methods contains a description of these invocations at the end of its documentation comment. The description begins with the phrase “This implementation.” This phrase should not be taken to indicate that the behavior may change from release to release. It connotes that the description concerns the inner workings of the method.